src/static/riot/competitions/tile/competition_tile.tag

```diff
@@ -1,104 +1,137 @@
-                        <div class="content">
-                            Tags: <b>Beginner</b>, <b>AutoML</b>
-                            <br>
-                            Admins: <b>tthomas63</b>
-                        </div>
-                    </div>
-                </div>
+            <div class="comp-info">
+                <h4 class="heading">
+                    {title}
+                </h4>
+                <p class="comp-description">
+                    {description}
+                </p>
+                <p class="organizer">
+                    <em>Organized by: <strong>{created_by}</strong></em>
+                </p>
             </div>
-            <div class="three wide blue column center aligned comp-tile-full-height">
-                <i>Comp deadline:</i>
-                <i>August 14 2018</i>
-                <div class="ui divider"></div>
-                <i>Phase deadline:</i>
-                <i>April 14 2018</i>
+            <div class="comp-stats">
+                {pretty_date(created_when)}
                 <div class="ui divider"></div>
-                <i>17 participants</i>
+                <strong>{participant_count}</strong> Participants
             </div>
         </div>
-    </div>
+    </a>
 
     <script>
+        var self = this
+
+        self.pretty_date = function (date_string) {
+            if (!!date_string) {
+                return luxon.DateTime.fromISO(date_string).toLocaleString(luxon.DateTime.DATE_FULL)
+            } else {
+                return ''
+            }
+        }
+
+        self.on('mount', () => {
+            $('.comp-description').each(function (f) {
+                var newstr = $(this).text().substring(0, 90);
+                if (newstr.length < 90) {
+                    $(this).text(newstr);
+                } else {
+                    $(this).text(newstr + '...')
+                }
+            })
+        })
+
+
     </script>
 
     <style type="text/stylus">
         :scope
             display block
-            margin-bottom 35px !important
+            margin-bottom 5px
+            cursor pointer
+
+        .link-no-deco
+            text-decoration none
+
+        .tile-wrapper
+            border solid 1px gainsboro
+            display inline-grid
+            grid-template-columns 0.1fr 3fr 1.3fr
+            min-width 425px
+            background-color #fff
+            transition all 75ms ease-in-out
+            color #909090
+            width 100%
 
-        .main-wrapper
-            border-left 5px solid #c7402d
-            border-top-left-radius 6px !important
-            border-bottom-left-radius 6px !important
+        .tile-wrapper:hover
+            box-shadow 0 3px 4px -1px #9c9c9c
+            transition all 75ms ease-in-out
+            background-color #e8e8e8
+            border solid 1px #b9b9b9
 
-        .comp-tile-paragraph
-            font-size 14px !important
+            .comp-stats
```

Comment:
Is this supposed to be indented?

Reply:
In Stylus, this is used to look for all child elements matching the class comp-stats in the tile-wrapper:hover class. Should be good!